mappings/net/minecraft/block/DyedCarpetBlock.mapping

```diff
@@ -1,6 +1,12 @@
 CLASS net/minecraft/class_5815 net/minecraft/block/DyedCarpetBlock
+	COMMENT A carpet that has an associated carpet color for {@linkplain net.minecraft.entity.passive.LlamaEntity llamas}.
```

Comment:
```suggestion
	COMMENT A carpet that has an associated dye color. The color is accessed by {@linkplain net.minecraft.entity.passive.LlamaEntity llamas}.
```

Reply:
I think this is a bit confusing since llamas have a carpet color specifically rather than dye color.

Reply:
Hmm, imo that color behavior is best documented in `LlamaEntity` class than here.